util/topsql/reporter/reporter.go

```diff
@@ -209,21 +209,94 @@ func (tsr *RemoteTopSQLReporter) Close() {
 	tsr.client.Close()
 }
 
+// collectedData stores the collected top-sql data.
+type collectedData struct {
+	// data stores topN records for each round.
+	data map[string]*dataPoints
+	// others stores the summary of all evicted records.
+	others *dataPoints
```

Comment:
I think you can use empty string as a key to store others, which is the same as TiKV. Then you don't need to change the structure:

```go
let keyOthers = ""
```